test/mpi.py

```diff
@@ -164,11 +164,27 @@ def test_is_mpirun_for(self):
 
     def test_set_omp_threads(self):
         """test if OMP_NUM_THREAD gets set correctly"""
+        del os.environ['OMP_NUM_THREADS']
+        del os.environ['OMP_PROC_BIND']
```

Comment:
This should be done conditionally, because if they're not defined, it'll break the test?

Reply:
yep

Reply:
@boegel fixed